### ISSUE TITLE
Missing values for parents wealth and income

### ISSUE THREAD

**Issue by dc0sic** [UI-Research/baby-bonds#1]
Many values for parents' wealth and income in 1997 are missing. The number of observations with missing wealth is 2327, those that are missing income is 2361 and those that are missing either one is 3106. 

To address this, we will impute these values using the multiple imputation method as was used by [Addo, Houle, and Simon (2016)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6049093/).

**Comment by judah-axelrod** [comment]
Damir, after generating the multiple implicates for parent net worth and income, how do they fit into the rest of the DYNASIM pipeline? Generally, the `mice` package includes functions for estimating models on each of the imputed datasets, pooling results together, and calculating standard errors. (E.g. running 10 regressions, calculating average regression coefficients across the 10 sets of coefficients, and calculating corresponding SEs). If these imputed data are next feeding into the hazard models, is that what we want to do here too?

**Comment by dc0sic** [comment]
Yes, that's one possible approach, use mice functions to estimate the model for each set of imputed values. However, we might be able to make our lives a little easier because we are less interested in standard errors and more in point estimates, because that's what we use in simulations. We can estimate our model(s) on multiple sets of imputations and compare the coefficients. If we get a wide range of estimates, we can average them as you suggested. However, if they fall close to each other, it might make sense to use a single set of estimates.

**Comment by dc0sic** [comment]
I just realized that we're not using the value of retirement savings in imputation. We should do that because we want to remove retirement savings from networth. Although, I am not sure how to do that. My guess is that we first need to impute the indicator variable for retirement savings and then impute the value of retirement savings only if the indicator is 1. I'll read more about it in the mice documentation. Just wanted to note it here.
